File: Iniciante/1010/1010.py
input_str = input()
c1, n1, v1 = input_str.split()
c1 = int(c1)
n1 = int(n1)
v1 = float(v1)

input_str = input()
c2, n2, v2 = input_str.split()
c2 = int(c2)
n2 = int(n2)
v2 = float(v2)

total = n1 * v1 + n2 * v2

print("VALOR A PAGAR: R$ %.2lf" %total)

File: Iniciante/Pagina 1/1010/1010.c
#include <stdio.h>

int main(){
    int c, n;
    double v, total;

    scanf("%d %d %lf", &c, &n, &v);
    total = n * v;
    scanf("%d %d %lf", &c, &n, &v);
    total += n * v;

    printf("VALOR A PAGAR: R$ %.2lf\n", total);

    return 0;
}